doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -138,7 +185,7 @@ recognised on input.
 
 =back
 
-=item C<ml-kem.output_formats> (B<OSSL_PKEY_PARAM_ML_KEM_INPUT_FORMATS>) <UTF8 string>
+=item C<ml-kem.output_formats> (B<OSSL_PKEY_PARAM_ML_KEM_OUTPUT_FORMATS>)
```

Comment:
?

Reply:
Was reverting commits to try to understand why CI was failing, all good now, there was a hard to spot typo in the fixup that changed a `2` to a `sizeof(uint32_t)` instead of `sizeof(uint16_t)` supposedly more clear.

Reply:
I have no idea what you're responding to here...

Reply:
The `?`, since the push was undoing a typo correction, and it seems you were questioning that.

Reply:
Er no.  There are three comments in the same general review, "Uhm... why are you frivolously removing the OSSL_PARAM data type indicators?"  This is one of them.

Reply:
... but, seems resolved, I think?  So ok